v-spec.adoc

```diff
@@ -1769,6 +1769,14 @@ address in the `rs1` field to byte offsets in vector register `vs2`.
                               #   and words from v3[i] to address x5+v5[i]+4
 ----
 
+For vector indexed segment loads, the destination vector register groups
+cannot overlap the source vector register group (specified by `vs2`), nor can
+they overlap the mask register if masked, else an illegal instruction
+exception is raised.
```

Comment:
Mask register constraint should be present for all vector segment loads, not just indexed.